client/payment-methods-map.js

```diff
@@ -98,4 +100,26 @@ export default {
 		currencies: [ 'EUR', 'PLN' ],
 		capability: 'p24_payments',
 	},
+	boleto: {
+		id: 'boleto',
+		label: __( 'Boleto', 'woocommerce-gateway-stripe' ),
+		description: __(
+			'Boleto is one of the most popular payment method in Brazil',
```

Comment:
Where does this description come from? Shouldn't it be "payment method**s**"?

I think the other descriptions here are based on Stripe docs: https://stripe.com/docs/payments/boleto

Reply:
What is your suggestion for a frase here? 

"Boleto is an official (regulated by the Central Bank of Brazil) payment method in Brazil." Also doesn't seem very informative. And coping the whole thing seems a bit too long.

Maybe "Boleto is an official payment method in Brazil. Customers receive a voucher that can be paid on banks, ATMs, or online bank portals. Payment confirmation takes 1 business day and funds are available after 2  business days after confirmation" would be better?

Reply:
I agree the first sentence from the Stripe docs for Boleto doesn't sound good.

I noticed most of the other descriptions combine the first two sentences from the Stripe docs, but aren't necessarily too descriptive. I don't think we need to inform how many business days the funds are available, etc. so I think we can cut that out:

> Boleto is an official payment method in Brazil. Customers receive a voucher that can be paid at authorized agencies or banks, ATMs, or online bank portals.

That is more or less the size of the P24 description.

Also, the current OXXO description doesn't look right to me. How about this?

> OXXO is a Mexican chain of convenience stores that allows customers to pay bills and online purchases in-store with cash.